### DISCUSSION TITLE
Cropper component: SelectionOptions AspectRatio Free

### DISCUSSION THREAD

**Discussion by DomenicoFormoso** [Megabit/Blazorise#5371] [General]
Can you set a free selection grid for the cropper component?
(without maintaining proportions)

**Comment by stsrki** [comment]
Yesr, just define the aspect ration as

```
SelectionOptions="@(new CropperSelectionOptions { AspectRatio = new(CropperAspectRatio.IsFree) })"
```

**Reply by stsrki** [reply]
Ps. you can investigate it in the demo source https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/CropperPage.razor

**Reply by DomenicoFormoso** [reply]
Thanks, I hadn't looked in the examples, now I've seen that you can do more things. Works well.
